components/playlist/browser/resources/components/playlistInfo.tsx

```diff
@@ -46,16 +46,20 @@ export default function PlaylistInfo ({
 }: Props) {
   return (
     <PlaylistInfoContainer className={className}>
-      <PlaylistName>
+      <PlaylistName className='playlist-name'>
         {isDefaultPlaylist ? 'Play Later' : playlistName}{' '}
       </PlaylistName>
-      <PlaylistDetail>{itemCount} items</PlaylistDetail>
+      <PlaylistDetail className='playlist-detail'>
+        {itemCount} items
+      </PlaylistDetail>
       <>{!!totalDuration && <PlaylistDetail>{totalDuration}</PlaylistDetail>}</>
```

Comment:
I don't think the fragment is doing anything here :smile:
```suggestion
      {!!totalDuration && <PlaylistDetail>{totalDuration}</PlaylistDetail>}
```